### PR TITLE
Fix: #2239 Quarkus k8s:resource automatic port pick up from application.properties

### LINKED ISSUE #2239: `k8s:resource` doesn't automatically pick up port from Quarkus `application.properties`
### Describe the bug

In a Quarkus project, when I specify port via application.properties:
```properties
quarkus.http.port=9000
```
`k8s:build` does pick up this port and exposes `9000` port in container image. However, `k8s:resource` doesn't seem to add `9000` port in generated Kubernetes liveness, readiness and startup probes.

If I add extra jkube-healthcheck-quarkus enricher configuration for port, it works:
```xml
      <plugin>
          <groupId>org.eclipse.jkube</groupId>
          <artifactId>kubernetes-maven-plugin</artifactId>
          <version>${jkube.version}</version>              
          <configuration>
            <enricher>
              <config>
                  <jkube-healthcheck-quarkus>
                      <port>9000</port>
                  </jkube-healthcheck-quarkus>
              </config>
            </enricher>  
          </configuration>
      </plugin>
```

### Eclipse JKube version

1.13.1

### Component

Kubernetes Maven Plugin

### Apache Maven version

None

### Gradle version

None

### Steps to reproduce

1. Enable health support in Quarkus project
2. Override default HTTP port via properties
3. Run `k8s:resource`, generated manifests contains probes containing `8080` port

### Expected behavior

Port should be automatically detected from properties while generating health checks

### Runtime

Kubernetes (vanilla)

### Kubernetes API Server version

1.25.3

### Environment

Linux

### Eclipse JKube Logs

_No response_

### Sample Reproducer Project

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by asingh2295**
## Description
k8s:resource doesn't automatically pick up port from Quarkus application.properties

Fixes #2239

## Type of change

 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [x] My code follows the style guidelines of this project
 - [ ] I Keep It Small and Simple: The smaller the PR is, the easier it is to review and have it merged
 - [ ] I use [conventional commits](https://www.conventionalcommits.org/) in my commit messages
 - [ ] I have performed a self-review of my code
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

File: jkube-kit/jkube-kit-quarkus/src/main/java/org/eclipse/jkube/quarkus/QuarkusUtils.java [hunk 1]
```diff
@@ -63,14 +63,7 @@ public class QuarkusUtils {
...
   private QuarkusUtils() {}
 
-  public static String extractPort(JavaProject javaProject, Properties properties, String defaultValue) {
-    final Optional<String> activeProfile = getActiveProfile(javaProject);
-    if (activeProfile.isPresent()) {
-      final String profilePort = properties.getProperty(String.format("%%%s.%s", activeProfile.get(), QUARKUS_HTTP_PORT));
-      if (StringUtils.isNotBlank(profilePort)) {
-        return profilePort;
-      }
-    }
```
Comment (rohanKanojia) [inline jkube-kit/jkube-kit-quarkus/src/main/java/org/eclipse/jkube/quarkus/QuarkusUtils.java:73]: Why have you removed this code from this method?

**Comment by rohanKanojia** [comment]
You should not modify existing tests. They are there to validate currently expected behavior.

**Comment by manusa** [comment]
I see we have 2 pull requests (PR) (#2291 and #2287) open for the same purpose. How are they different? if one of them is redundant, please close it.

**Comment by asingh2295** [comment]
I'm closing this one, need some discussion before merging. 

https://github.com/eclipse/jkube/pull/2287
We can merge this one as fix.

**Pull request closed without merging**